docs/Project-Configuration/Commit-Queue.md

```diff
@@ -71,6 +71,7 @@ Pass in either a single commit hash, or a range `<commit_hash1>..<commit_hash2>`
 * `--force` 
 
 This flag forces the patch to the front of the queue. Use this sparingly and only for urgent scenarios, such as an urgent revert.
+This flag does not work when used in conjunction with the `--pause` flag and should be used with the `--resume` flag.
```

Comment:
Why is "resume" needed? Does it only work with "resume"?

Reply:
hmm its like 
`evergreen commit-queue merge --force --pause` 
`evergreen commit-queue merge --resume` wont work


`evergreen commit-queue merge --pause` 
`evergreen commit-queue merge --force --resume` will work  


ill try to make it clearer